rrmngmnt/host.py

```diff
@@ -320,6 +320,19 @@ def service(self, name, timeout=None):
             service = self._create_service(name, timeout)
             self._service_provider = service.__class__
             return service
+        
+    def get_service_status(self, service_name):
```

Comment:
I prefer to do not put methods under host class, if we will continue in this manner we will end up as we did with utilities/machine.py class ... where we had 100+ methods for different purpose ...
I prefer to keep methods under their modules and Host is just entrypoint to dispatch them.

Reply:
Fixed